src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -16,9 +16,20 @@
       Overwrite="true"
       Encoding="ASCII" />
 
+    <GetDependencyInfo 
```

Comment:
In the same spirit, can WinForms version to be retreived and placed in this file? I would guess that SDK/WPF and WindowsDesktop versions maybe interested to somebody.